app/views/other/add/DescriptionView.scala.html

```diff
@@ -0,0 +1,48 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.other.add.routes._
+
+@this(
+    main_template: MainTemplate,
+    formHelper: FormWithCSRF
+)
+
+@(form: Form[_])(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = s"${errorPrefix(form)} ${messages("otherBeneficiary.description.title")}"
+    ) {
+
+    @formHelper(action = DescriptionController.onSubmit(), 'autoComplete -> "off") {
+
+        @components.back_link()
+
+        @components.error_summary(form.errors)
+
+        @components.input_text(
+            field = form("value"),
+            label = messages("otherBeneficiary.description.heading"),
+            headingIsLabel = true,
+            hint = Some(messages("otherBeneficiary.description.hint")),
+            labelClass=Some("heading-large")
```

Comment:
Add `inputClass = Some("form-control-3-4")` to this to make the text box wider

Reply:
Done